pkg/defaults/defaults.go

```diff
@@ -48,8 +48,8 @@ const (
 	DefaultRegistryPort = 5000
 
 	//tags, versions, repos
-	DefaultEVETag               = "0.0.0-master-41772163" //DefaultEVETag tag for EVE image
-	DefaultAdamTag              = "0b72a46462c4e1e7de5acce00cea076feb3f8585"
+	DefaultEVETag               = "0.0.0-master-f989c3c5" //DefaultEVETag tag for EVE image
```

Comment:
Let's not use EDEN with master tags. Point it to release versions.

Reply:
Done